dcm4che-net/src/main/java/org/dcm4che3/net/service/DicomServiceRegistry.java

```diff
@@ -85,7 +85,7 @@ public void onDimseRQ(Association as, PresentationContext pc,
         try {
             lookupService(as, dimse, cmd).onDimseRQ(as, pc, dimse, cmd, data);
         } catch (DicomServiceException e) {
-            Association.LOG.info("{}: processing {} failed. Caused by:\t",
+            Association.LOG.error("{}: processing {} failed. Caused by:\t",
```

Comment:
The Exception may be caused by a bad request => prefer to stay with INFO level.

Reply:
While it may be caused by a bad request, often we are asked to look at why we reject/return an error as a start point to finding the bad request. In some sites the info/warn level logs are rolling out of the logs before we are able to collect the logs resulting in delays in diagnosing the problems while at error level the logs will be retained longer. Additionally, error level is consistent with a non-recoverable error condition vs an informational message.

However, I will return the log level to info should you feel strongly about it.

Reply:
That argument would lend toward the need to add auditing as it is a non-recoverable processing error. Changing the log level doesn't stop that and the lower level does make the diagnosis of the problem more difficult. 

However, I have reverted the log level as per your request.